sdk/test/opentelemetry/sdk/trace/export/in_memory_span_exporter_test.rb

```diff
@@ -7,4 +7,67 @@
 require 'test_helper'
 
 describe OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter do
+  export = OpenTelemetry::SDK::Trace::Export
+
+  let(:span1)   { OpenTelemetry::Trace::Span.new }
+  let(:span2)   { OpenTelemetry::Trace::Span.new }
+
+  let(:subject) { export::InMemorySpanExporter.new }
+
+  it 'accepts an Array of Spans as argument to #export' do
+    subject.export([span1, span2])
```

Comment:
While this *does* work, thanks to the wonders of dynamic typing, it’s technically incorrect. We should be exporting `SpanData`:
```ruby
subject.export([span1.to_span_data, span2.to_span_data])
```
Of course, that makes the expectations a little more difficult to express...

Reply:
Same concern throughout these tests.

Reply:
I was working from the doc comments on the method under test:

```          # Called to export sampled {Span}s.
          #
          # @param [Enumerable<Span>] spans the list of sampled {Span}s to be
          #   exported.
          # @return [Integer] the result of the export, SUCCESS or
          #   FAILED_NOT_RETRYABLE
          def export(spans)
```

https://github.com/open-telemetry/opentelemetry-ruby/blob/master/sdk/lib/opentelemetry/sdk/trace/export/in_memory_span_exporter.rb#L59

So these comments need switching to refer to `SpanData`s not `Span`s too?

Reply:
Yep - comments are wrong.